tuxemon/core/item.py

```diff
@@ -211,6 +212,11 @@ def heal(self, user, target):
         if target.current_hp == target.hp:
             return {"success": False}
 
+        # don't heal if fainted
+        print(check_status(target, 'status_faint'))
```

Comment:
please remove this print statement.  you can change it to a logger if you would like to keep it.

Reply:
ill remove it